client/scripts/views/components/new_proposal_button.ts

```diff
@@ -48,6 +52,21 @@ export const getNewProposalMenu = (candidates?: Array<[SubstrateAccount, number]
       label: 'New text proposal',
       iconLeft: mobile ? Icons.PLUS : undefined,
     }),
+    app.chain?.base === ChainBase.Ethereum && app.chain?.network === ChainNetwork.Aave
```

Comment:
Why no section for dydx?

Reply:
We may have to follow up on this one, right now we only have ChainNetwork.AAVE (no dydx), I didn't go through and add dydx as a chain.